schema/errors.go

```diff
@@ -0,0 +1,20 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import "errors"
+
+// errInvalidProtoIsNil is returned when trying to convert a proto schema to a schema object and the proto is nil.
```

Comment:
Actually it's the other way around, `ToProto` converts from schema to proto object.

Reply:
@lovromazgon good catch 🤦